docs/en/observability/monitor-java-app.asciidoc

```diff
@@ -0,0 +1,2005 @@
+          "names": ["heartbeat-*"],
+          "privileges": ["view_index_metadata", "create_doc"]
+        }
+      ]
+    }
+  }
+}
+----
+
+Let’s setup the {heartbeat} keystore and run the setup.
+
+[source,bash]
+----
+./heartbeat keystore create
+echo -n "observability-javalin-app:ZXUtY2VudHJhbC0xLmF3cy5jbG91ZC5lcy5pbyQ4NDU5M2I1YmQzYTY0N2NhYjA2MWQ3NTJhZWFhNWEzYyQzYmQwMWE2OTQ2MmQ0N2ExYjdhYTkwMzI0YjJiOTMyYQ==" | ./heartbeat keystore add CLOUD_ID --stdin
+echo -n "SCdUSHMB1JmLUFPLgWAY:R3PQzBWW3faJT01wxXD6uw" | ./heartbeat keystore add ES_API_KEY --stdin
+
+./heartbeat setup -e -E 'cloud.id=${CLOUD_ID}' -E 'cloud.auth=elastic:YOUR_SUPER_SECRET_PASS'
+----
+
+Add some services to monitor.
+
+[source,yaml]
+----
+name: heartbeat-shipper
+
+cloud.id: ${CLOUD_ID}
+output.elasticsearch:
+  api_key: ${ES_API_KEY}
+
+heartbeat.monitors:
+  - type: http
+    id: javalin-http-app
+    name: "Javalin Web Application"
+    urls: ["http://localhost:7000"]
+    check.response.status: [200]
+    schedule: '@every 15s'
+
+  - type: http
+    id: httpbin-get
+    name: "httpbin GET"
+    urls: ["https://httpbin.org/get"]
+    check.response.status: [200]
+    schedule: '@every 15s'
+
+  - type: tcp
+    id: javalin-tcp
+    name: "TCP Port 7000"
+    hosts: ["localhost:7000"]
+    schedule: '@every 15s'
+
+processors:
+  - add_observer_metadata:
+      geo:
+        name: europe-munich
+        location: "48.138791, 11.583030"
+----
+
+Now start {heartbeat} and wait a couple of minutes to get some data. To view the Uptime app, 
+select *Observability* -> *Uptime*. The overview looks
+like this.
+
+image:./images/monitor-java-app-uptime-overview.png[Uptime Overview]
+
+You can see the list of monitors and a global overview. Let’s see the
+details for one of those alerts. Click *Javalin Web Application*.
+
+You can see the execution for the last
+scheduled checks, but the duration for each check might be more interesting.
+You can see if the latency for one of your checks is going up. 
+
+The interesting part is the world map at the top. You can specify in the
+configuration where the check originated, which in this case was in Munich in Europe.
+By configuring several {heartbeat}s running across the world, you
+can compare latencies and figure out which data center you need to run your
+application to be next to your users.
+
+The duration of the monitor is in the low milliseconds, as it is really
+fast. Check the monitor for the `httpbin.org` endpoint, and you will see a
+much higher duration. In this case, it is about 400ms for each request. This
+is not too surprising, because the endpoint is not nearby and you need to
+initiate a TLS connection for every request, which is costly.
+
+Do not underestimate the importance of this kind of
+monitoring. Also, consider this just the beginning as the next step is
+to have synthetics that monitor the correct behavior of your
+application, for example, to ensure that your checkout process works all
+the time.
```

Comment:
Can you add a "What's next" section that points users to the next great thing they want to do?